docs/operation/Recover_From_Etcd_Permanent_Quorum_Loss.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Recover from permanent quorum loss of ETCD cluster in shoot control plane
+---
+
+## Potential Situation
+Majority of ETCD pods(Greater than or equal to n/2 + 1) are down in the ETCD cluster (main or events) in the shoot control plain.
+
+## Quorum loss in ETCD Cluster
+[Quorum loss](https://etcd.io/docs/v3.3/op-guide/recovery/) means when majority of ETCD pods(Greater than or equal to n/2 + 1) are down simultaneously for some reason. 
+
+There are two types of quorum loss that can happen to ETCD multinode cluster -
+
+1. **Transient quorum loss** - A quorum loss is called transient when majority of ETCD pods are down simultaneously for some time. The pods may be down for network unavailability, high resource usages etc. When the pods come back after some time, they can re-join to the cluster and the quorum is recovered automatically without any manual intervention. The data directory must not be corrupted for majority of pods in this type of quorum loss.
+
+2. **Permanent quorum loss** - A quorum loss is called permanent when majority of ETCD pods are down simultaneously due to unavailability of data volumes that are/were attached to the ETCD pods to hold the ETCD data. If majority of ETCD data volumes are unavailable **SIMULTANEOUSLY**, the etcd cluster is not going to recover automatically from the quorum loss. A human operator will now need to intervene and execute the following steps to start a fresh multi-node ETCD cluster.
+
+## ETCD clusters in shoot control plane
+There are two [ETCD clusters](https://github.com/gardener/etcd-druid/tree/master/docs/proposals/multi-node) running in shoot control plane. One is named as ETCD Events and another is named as ETCD main. The operator needs to take care of permanent quorum loss to a specific cluster. If permanent quorum loss occurs to ETCD events cluster, the operator needs to note down the PVCs, configmaps, statefulsets, CRs etc related to ETCD events cluster and work on those resources only. Following steps guide a human operator to recover from permanent quorum loss of **ETCD main cluster**. 
```

Comment:
Please make the playbook generic enough for any `etcd-druid` user to use. Gardener uses `etcd-main` and `etcd-events`, but a community member using `etcd-druid` by itself can create ETCDs with any name like `etcd-test` or `my-etcd`. Please remove the gardener-specific details from this doc.